bindings/experimental/distrdf/python/DistRDF/Backends/Spark/__init__.py

```diff
@@ -0,0 +1,10 @@
+def RDataFrame(*args, **kwargs):
+    """
+    Create a SparkDataFrame object
```

Comment:
This is actually not true anymore, since we will return a `DataFrame`, right?

Reply:
yes, I will correct the docs